pages/index.js

```diff
@@ -1,63 +1,165 @@
+    event.preventDefault();
+    if (moodLevel <= 3) {
+      setCurrentQuote(
+        motivationalQuotesBad[getRandomIndex(motivationalQuotesBad.length)]
+      );
+      setMoodMode("Feeling off today? Don’t sweat it, you’ll bounce back!");
+    } else if (moodLevel <= 7 && moodLevel >= 4) {
+      setCurrentQuote(
+        motivationalQuotesOkay[getRandomIndex(motivationalQuotesOkay.length)]
+      );
+      setMoodMode("A bit off today? Every step forward is still a win!");
+    } else if (moodLevel >= 8) {
+      setCurrentQuote(
+        motivationalQuotesGood[getRandomIndex(motivationalQuotesGood.length)]
+      );
+      setMoodMode("On top of the world today? Keep crushing it!");
+    }
   }
 
   return (
-    <StyledFlexWrapper>
-      <ExerciseList>
-        <StyledFilterWrapper>
-          <StyledFilterButton onClick={() => setFilterMode(!filterMode)}>
-            {filterMode ? "Filter ▲" : "Filter ▼"}
-          </StyledFilterButton>
-          {filterMode ? (
-            <FilterSection
-              filters={filters}
-              onAddFilter={onAddFilter}
-              handleDisableFilter={handleDisableFilter}
-              handleClear={handleClear}
+    <QuoteContainer>
+      <StyledHeadline>
+        {moodMode ? moodMode : "Hello, how are you feeling today?"}
+      </StyledHeadline>
+      {moodMode ? (
+        <>
+          <StyledPostionRelative>
+            <StyledQuotationMarks
+              src={"/quote.svg"}
+              width={100}
+              height={100}
+              alt="quote decoration"
             />
-          ) : null}
-        </StyledFilterWrapper>
-        {!filterExercises(exercises, filters).length ? (
-          <p>No exercises found. Please search for different muscle groups.</p>
-        ) : null}
-        {filterExercises(exercises, filters).map((exercise) => (
-          <Exercise key={exercise.id} exercise={exercise} />
-        ))}
-      </ExerciseList>
-    </StyledFlexWrapper>
+          </StyledPostionRelative>
+          <QuoteText>{currentQuote.quote}</QuoteText>
+          <AuthorText>{currentQuote.author}</AuthorText>
+        </>
+      ) : (
+        <>
+          <form
+            onSubmit={(event) => handleMood(event, event.target.mood.value)}
+          >
+            <label htmlFor="mood">My mood today is...</label>
+            <br />
+            <br />
+            <span>😔</span>
+            <input type="range" min={1} max={10} id="mood" name="mood"></input>
+            <span>🤩</span>
+            <br />
+            <Button type="submit">Send</Button>
+          </form>
+        </>
+      )}
+    </QuoteContainer>
   );
 }
+
+const StyledHeadline = styled.h2`
+  font-size: 1.5rem;
+  margin: 0;
+  margin-bottom: 1rem;
+`;
+
+const StyledQuotationMarks = styled(Image)`
+  position: absolute;
+  right: 80px;
+  top: -30px;
+`;
+
+const StyledPostionRelative = styled.div`
+  position: relative;
+`;
+
+const QuoteContainer = styled.div`
```

Comment:
This could be an article as the whole quote could be standing alone as well.

The appropiate element for a quote is [`<blockquote>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote).

I would also recommend using CSS for layout sections instead of relying on `<br>` elements. 🙂